shared/postgres/compose.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  benchi-postgres:
+    container_name: benchi-postgres
+    networks:
+      - benchi
+
+    image: quay.io/debezium/postgres:17
```

Comment:
Btw, what's the reason behind using the debezium image over the [official one](https://hub.docker.com/_/postgres)?

Reply:
From the readme:

> This image is based upon [postgres:11-apline](https://hub.docker.com/_/postgres/) and adds two logical decoding plug-ins:
> 
> - [postgres-decoderbufs⁠](https://github.com/debezium/), based on Protocol Buffers and maintained by the Debezium community
> - [wal2json⁠](https://github.com/eulerto/wal2json), based on JSON and maintained by the wal2json community
> 
> Both are supported by the [Debezium PostgreSQL Connector⁠](http://debezium.io/docs/connectors/postgresql/) to capture changes committed to the database and record the data change events in Kafka topics. In addition, Debezium supports the pgoutput plug-in, which is available by default on Postgres 10 and later.

Do we need these plug-ins? If not, I'd just switch to the official image.

Reply:
@lovromazgon (no reason) good call.